masterlist.yaml

```
@@ -2799,6 +2799,23 @@ plugins:
         util: '[TES4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 7
 
+  - name: '(One of a Kind Weapons|OoaKW - Staff of Worms)?\.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/47749/' ]
+    tag: [ Graphics ]
+  - name: 'One of a Kind Weapons.esp'
+    clean:
+      - crc: 0x3CFE1D45
+        util: 'TES4Edit v4.0.4'
+      - crc: 0xBF48B2BE
+        util: 'TES4Edit v4.0.4'
```

Both versions of **One of a Kind Weapons.esp** contain a wild edit, so a message can be added instead of cleaning data.

```suggestion
    msg:
      - <<: *wildEdits
        subs: [ 'Remove Creature **000855A4** from **One of a Kind Weapons.esp**.' ]
        condition: 'checksum("One of a Kind Weapons.esp", 3CFE1D45) or checksum("One of a Kind Weapons.esp", BF48B2BE)'
```